source/operations/concepts/healing.rst

```diff
@@ -0,0 +1,114 @@
+
+If all of the data shards are found intact, MinIO serves the object from the data shards without inspecting the corresponding parity shards.
+
+If the object has missing or damaged data shards, MinIO uses the available parity shards to heal the object before serving it as part of the operation.
+There **must** be an intact parity shard available for each lost or damaged data shard, otherwise the object cannot be recovered.
+If any parity shards are lost or damaged, MinIO restores the parity shard, provided there are sufficient other parity shards to serve the object.
+
+Healing with the object scanner
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO uses an :ref:`object scanner <minio-concepts-scanner>` to perform a number of tasks related to objects.
+One of these tasks checks the integrity of objects and, if found damaged or corrupted, heals them.
+
+On each scanning pass, MinIO uses a hash of the object name to select one out of every 1,024 objects to check.
+
+If any object is found to have lost data shards, MinIO heals the object from available parity shards.
+By default, MinIO does *not* check for :term:`bit rot` corruption using the scanner.
+This can be an expensive operation to perform and the risk of bit rot across multiple disks is low.
+
+Healing by manual request
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Administrators can use :mc:`mc admin heal` to initiate a full system healing.
+The procedure is very resource intensive and not typically needed.
+
+Consult with MinIO Engineers before manually starting a healing process on a deployment.
+
+Healing metrics
+---------------
+
+MinIO provides several `healing metrics <https://min.io/docs/minio/linux/operations/monitoring/metrics-and-alerts.html#healing-metrics>`__ to monitor the status of healing processes on a deployment.
+
+Refer to the :ref:`minio-metrics-and-alerts` for more information on available endpoints and configuration.
+
+.. _minio-concepts-healing-colors:
+
+Healing Output Color Key
+------------------------
+
+Earlier MinIO versions generated a table with green, yellow, red, and gray colors to indicate the healing status of individual objects.
+These colors have been deprecated in favor of more detailed :ref:`healing metrics available at the cluster level <minio-metrics-and-alerts-available-metrics>`.
+
+The following table describes the meaning of the now-deprecated status colors.
+
+.. list-table::
+   :widths: 25 75
+   :width: 100%
+
+   * - **Green**
+     - *Healthy*, the object has all data and parity shards available as required to serve the object
+ 
+   * - **Yellow** 
+     - *Healing*, the object is still in the process of healing, and there are sufficient data or parity shards available to complete the healing
+
+   * - **Red** 
+     - *Unhealthy*, the object has lost one or more shards and requires healing
+
+   * - **Grey** 
+     -  *Unrecoverable*, the object has lost too many data and/or parity shards and cannot be healed or recovered
+
```

Comment:
Does all of this matter if it is deprecated?

Reply:
I think it comes up often enough in support cases, and we have some UI/UX in Console/Enterprise Console that also uses color coding.

Reply:
As @ravindk89 says, this is mainly for those times when someone comes across it in an older version or in the Console and wants to know what the colors mean. This still happens.